PokemonRampUpTests/PokemonModelTests.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  PokemonRampUpTests.swift
+//  PokemonRampUpTests
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import XCTest
+@testable import PokemonRampUp
+
+// TO-DO [POKEMON-0011] Create unit test for Pokemon Model
+final class PokemonModelTests: XCTestCase {
+
+    // NOTE: This test is to sure the Pokemon Model works as intended. Will come back later in TO-DO [POKEMON-0011] to complete all unit tests for this model
+    func testSuccessfullyPokemonModelDecoding() {
```

Comment:
testMethodNameWithCircumstancesShouldExpectation

Reply:
Since youre following given, when, then. I would follow the same convention when naming your unit test method.

Reply:
I updated it with this naming convention👍